eng/targets/ResolveReferences.targets

```diff
@@ -11,15 +11,16 @@
 
   Items used by the resolution strategy:
 
-    * BaselinePackageReference = a list of packages that were reference in the last release of the project currently building
+    * BaselinePackageReference = a list of packages that were referenced in the last release of the project currently building
```

Comment:
Probably easiest to review changes to this file on a per-commit basis for about the last 5 commits.